includes/index.php

```diff
@@ -0,0 +1 @@
+<?php // Silence is golden
```

Comment:
Remove if this is not needful

Reply:
added this file so that main files are not accessible directly.